packages/framework/agent-scheduler/src/scheduler.ts

```diff
@@ -387,20 +387,22 @@ export class AgentScheduler extends TypedEventEmitter<IAgentSchedulerEvents> imp
 }
 
 class AgentSchedulerRuntime extends FluidDataStoreRuntime {
```

Comment:
I wonder what our stance is overall RE how data stores are extended. Are we supporting / pushing people toward any specific model (inheritance like here, extension points like with DataObject)?

Reply:
In general, i prefer composition over inheritance. i don't think we have a strong perspective. some tradeoffs i would want to understand further are: testability, and reusability.